src/lib/mongoBackend/MongoGlobal.h

```diff
@@ -365,7 +365,14 @@ extern BSONArray processConditionVector
 * processAvailabilitySubscriptions -
 *
 */
-extern bool processAvailabilitySubscription(EntityIdVector enV, AttributeList attrL, std::string subId, std::string notifyUrl, Format format, std::string tenant);
+extern bool processAvailabilitySubscription(
+    EntityIdVector     enV,
+    AttributeList      attrL,
```

Comment:
Of course... but that is part of a next PR. The "strategy" is going PR by PR with small steps (due to each change involves many files).

NTC

(I love copy-paste :)